Website/src/items/templates/edgewebsite/HeroSectionCta.yml

```diff
@@ -1,8 +1,8 @@
 ﻿---
-ID: "b33a6d2b-aaa4-567b-8640-18c9e613435e"
+ID: "76d550fa-6541-4c83-9f39-7b7e5b4cba62"
 Parent: "59c21fbf-d36e-5462-9847-c8c0bf7786ab"
 Template: "ab86861a-6030-46c5-b394-e8f99e8b87db"
-Path: /sitecore/templates/Project/edgewebsite/HeroSection
+Path: /sitecore/templates/Project/edgewebsite/HeroSectionCta
```

Comment:
Since templates were changed, you must run `pre-pr-updates.ps1` to regenerate the GraphQL introspection data. This must be done with a running local Sitecore instance.

Reply:
100% forgot about this. Thanks!